src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx

```diff
@@ -10,21 +11,59 @@ const displayName = 'common.Extensions.UserNavigation.partials.UserSubmenu';
 
 const UserSubmenu: FC<UserSubmenuProps> = ({ submenuId }) => {
   const isMobile = useMobile();
-  const iconSize = isMobile ? 'small' : 'extraSmall';
+  const iconSize = isMobile ? 'small' : 'tiny';
+
+  const handleMenuItemClick = (
+    event: React.MouseEvent<HTMLAnchorElement>,
+    onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void,
+  ) => {
+    if (onClick) {
+      event.preventDefault();
+      onClick(event);
+    }
+  };
 
   return (
     <ul className="-mb-2">
-      {userSubmenuItems[submenuId].map(({ id, url, icon, label }) => (
+      {userSubmenuItems[submenuId].map((item) => (
         <li
-          key={id}
-          className="mb-2 last:mb-0 sm:mb-0 hover:bg-gray-50 rounded -ml-4 w-[calc(100%+2rem)]"
+          key={item.id}
+          className={clsx(
+            'mb-2 last:mb-0 sm:mb-0 hover:bg-gray-50 rounded -ml-4 w-[calc(100%+2rem)]',
+            item.className,
+          )}
         >
-          <NavLink to={url} className="flex items-center navigation-link">
-            <span className="flex items-center shrink-0 mr-2 sm:mr-0 flex-grow">
-              <Icon name={icon} appearance={{ size: iconSize }} />
-              <p className="ml-2 ">{label}</p>
-            </span>
-          </NavLink>
+          {item.external ? (
+            <a
+              href={item.url}
+              className={clsx(
+                'flex items-center navigation-link',
+                item.className,
+              )}
+              target="_blank"
+              rel="noopener noreferrer"
+              onClick={item.onClick}
+            >
+              <span className="flex items-center shrink-0 mr-2 sm:mr-0 flex-grow">
+                <Icon name={item.icon} appearance={{ size: iconSize }} />
+                <p className="ml-2 ">{item.label}</p>
+              </span>
+            </a>
+          ) : (
+            <NavLink
```

Comment:
This doesn't need to be a NavLink if we're not really using it, do we? Or is this avout the `active` class?

Reply:
This is likely going to be needed in future.

Reply:
What do you mean by "this"? Can you expand please?

Reply:
probably gonna link somewhere inside the app at some point? dunno, we can keep it like this and realign when we get there. I'd like to have just a "button" component here too, but this PR is getting old and it's constantly getting new conflicts :')

Reply:
Yeah, well I really don't like this kind of stuff, it gets out of hand quickly. But if you create an issue for it we can leave it like this.
A button is a button and a link is a link :)